docs/learn/integration-tutorials/.gitignore

```diff
@@ -1 +0,0 @@
-target/
```

Comment:
Do we need to keep _target_ directory?

Reply:
This .gitignore file is not needed since `integration-tutorials` is not a Ballerina project.